packages/core-interop/src/core-interop/Logging.ts

```diff
@@ -3,20 +3,43 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-import { Logger as CoreLogger, LogLevel as CoreLogLevel } from "@itwin/core-bentley";
+import { LogLevel as CoreLogLevel } from "@itwin/core-bentley";
 import { ILogger, LogLevel } from "@itwin/presentation-hierarchies";
 
 /**
- * Create an `ILogger` that uses [Logger]($core-bentley) API to log messages.
+ * Defines input for `createLogger`. Generally, this is the [Logger](https://www.itwinjs.org/reference/core-bentley/logging/logger/)
+ * class from `@itwin/core-bentley` package.
+ */
+interface ICoreLogger {
+  isEnabled(category: string, level: CoreLogLevel): boolean;
+  logError(category: string, message: string): void;
+  logWarning(category: string, message: string): void;
+  logInfo(category: string, message: string): void;
+  logTrace(category: string, message: string): void;
+}
+
+/**
+ * Create an `ILogger` that uses [Logger](https://www.itwinjs.org/reference/core-bentley/logging/logger/) API to log messages.
```

Comment:
In most cases you wrote "defines" or "registers", etc. Here (and in other cases below) you wrote "create". Is this intentional?

Reply:
No, I just unified the wording. Thanks for noticing.